### PR TITLE
Add Wix Style React lint plugin to react rules

### PR DESCRIPTION
**Pull request by yairhaimo**

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -11,13 +11,14 @@ module.exports = {
...
+  plugins: ['lodash', 'eslint-plugin-wix-style-react'],
   rules: {
     'lodash/import-scope': 'warn',
     'react/jsx-no-bind': [2, {allowArrowFunctions: true}],
     'react/jsx-pascal-case': 0,
     'react/no-unused-prop-types': 0,
     'react/no-children-prop': 0,
-    'react/forbid-component-props': 0
+    'react/forbid-component-props': 0,
+    'wix-style-react/no-full-wsr-lib': 2
```
Comment (ranyitz) [inline react.js]: forbid-import-whole-wsr ?

Comment (ranyitz) [reply react.js]: or `wsr-only-internal-modules` as opposed to https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-internal-modules.md

Comment (ranyitz) [reply react.js]: just a suggestion though 😉

Comment (yairhaimo) [reply react.js]: i went with the convention of `no-...`
its a temporary rule anyway 🤷‍♂️

Comment (ranyitz) [reply react.js]: Sound good 👍

**Pull request merged**